### ISSUE TITLE
Zen Nav Questions

### ISSUE THREAD

**Issue by beccaklam** [mozilla/foundation.mozilla.org#3077]
Mavis noticed that she was having trouble aligning things for the blog design because the nav is not centered. Currently, abstract has the nav slightly off center on zen nav where the menu sticks out:
<img width="653" alt="Screen Shot 2019-04-29 at 4 37 34 PM" src="https://user-images.githubusercontent.com/20977376/56927226-a4486e80-6aa1-11e9-99c6-7ae6b1934df3.png">

I wasn't aware of this and designed the tablet zen nav to be centered:
<img width="812" alt="Screen Shot 2019-04-29 at 4 38 41 PM" src="https://user-images.githubusercontent.com/20977376/56927244-af9b9a00-6aa1-11e9-8f78-caf64a84b43f.png">

The mobile nav is centered: 
<img width="341" alt="Screen Shot 2019-04-29 at 5 11 27 PM" src="https://user-images.githubusercontent.com/20977376/56927304-d954c100-6aa1-11e9-9bc2-553c75e67a95.png">

**To discuss:** when should the zen nav be centered and when should the menu 'stick out'?
@kristinashu @sabrinang @natalieworth

**Comment by alanmoo** [comment]
This sounds vaguely familiar- it may have been because we had more items in the main nav and they wrapped really strangely on tablet view, so the menu button was popped to the left a bit to accommodate.

**Comment by kristinashu** [comment]
I think it is supposed to stick out from the grid so that the logo would be in the same spot whether the hamburger was there or not. 

<img width="899" alt="image" src="https://user-images.githubusercontent.com/1068783/56975139-05655600-6b25-11e9-84e8-a3c11fcd52c5.png">

But then that is not possible on mobile/tablet in order to maximize the grid space and reduce the space on the left/right sides.

Side note: when Scott made this originally, we tried to delay hiding nav items in a hamburger for as long as possible, for better UX. This was the example I gave him https://codepen.io/chriscoyier/pen/bppOVP/. It never quite worked out well and looks like we lost that functionality at some point along the way. But that _might_ be what you were thinking of Alan.
